test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -202,13 +324,10 @@ public function invalidVersionStringsProvider() : array
     {
         return [
             [''],
-            ['12.a'],
-            ['12a3'],
             ['alpha'],
             ['beta'],
-            ['1-a'],
-            ['1.2.a'],
```

Comment:
`1.2.a` is invalid IMO: should probably still be there

Reply:
sorry, why is this invalid? I'm confused

Reply:
`1.2-alpha` seems valid - `1.2.alpha` is unlikely so

Reply:
ok, I tackled this by requiring dash `-` in front of stability flag